kano_settings/system/audio.py

```diff
@@ -99,13 +99,13 @@ def is_HDMI():
 
     if amixer_string.find(hdmi_string) != -1:
         # Make sure config file is up to date
-        if get_setting('Audio') != 'HDMI':
-            set_setting('Audio', "HDMI")
+        if get_setting('Audio') != _("HDMI"):
+            set_setting('Audio', _("HDMI"))
```

Comment:
Turns out this is not so much of a problem because `json` knows how to handle `unicode` and it's default encoding is `utf8` - [doc here](). Under `en_QQ` setting the audio to `HDMI` produces the following file:
```
radu@radu ~/kano-settings $ cat /usr/share/kano-settings/config/settings 
{
  "Audio": "\uff28\uff24\uff2d\uff29"
}
```
cc @tombettany